oceanprotocol/ocean.py
# Fix c2d README

Fixes #624.

Changes proposed in this PR:

- Add the README `test_c2d_flow_readme` integration test
  - Retrieving the final algorithm output is blocked until the c2d backend supports v4, so this PR skips the newly added `test_c2d_flow_readme` integration test.
- Overhaul the c2d-flow.md README
- Update tox to use barge `v4_c2d` branch and run barge `--with-c2d`
- Remove `READMEs/ALG_ddo.md` and `READMEs/DATA_ddo.md`
- Fix minor logic error in `DataServiceProvider.initialize()` where `compute_data=""` would not be forwarded to provider, when it should be.
- Fix `OceanAssets.update` return docstring
- Fix typos and formatting in `test_marketplace_flow` and `marketplace-flow.md`
- Fix `get_registered_asset_with_compute_service` so it doesn't add old compute keys like "cpus" or "gpus".
